dbt/adapters/tidb/impl.py

```diff
@@ -277,3 +282,37 @@ def get_rows_different_sql(
         )
 
         return sql
+
+    def valid_incremental_strategies(self):
+        """The set of standard builtin strategies which this adapter supports out-of-the-box.
+        Not used to validate custom strategies defined by end users.
+        """
+        return ["delete+insert", "append"]
+
+    @available.parse_none
+    def get_incremental_strategy_macro(self, model_context, strategy: str):
```

Comment:
Can we not override this method and use the dbt implement? I don't see we need to override this according to the https://github.com/dbt-labs/dbt-core/discussions/6011